proto/interchain_security/ccv/provider/v1/tx.proto

```diff
@@ -88,4 +89,24 @@ message MsgOptOut {
   string provider_addr = 2 [ (gogoproto.moretags) = "yaml:\"address\"" ];
 }
 
-message MsgOptOutResponse {}
\ No newline at end of file
+message MsgOptOutResponse {}
+
+// MsgSetConsumerCommissionRate allows validators to set 
+// a per-consumer chain commission rate
+message MsgSetConsumerCommissionRate {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+  // The chain id of the consumer chain to set a commission rate
+  string chain_id = 1;
+   // The validator on the provider
+   string provider_addr = 2 [ (gogoproto.moretags) = "yaml:\"address\"" ];
+   // The rate to charge delegators on the consumer chain, as a fraction
+   string rate = 3 [
```

Comment:
Would it make sense to call this `commission_rate_fraction`?

Reply:
It would, but I'd prefer to keep the naming aligned with the SDK code and not too long.